src/puppetlabs/trapperkeeper/config.clj

```diff
@@ -59,6 +59,12 @@
     #{".yaml" ".yml"}
     (yaml/parse-string (slurp file))))
 
+(defn override-restart-file-from-cli-data
+  [config-data cli-data]
+  (if-let [cli-restart-file (:restart-file cli-data)]
+    (assoc-in config-data [:global :restart-file] cli-restart-file)
```

Comment:
Yeah, it would be logged in both places.  Not great, but not the end of the world, I suppose.